integration_tests/models/condition.sql

```diff
@@ -0,0 +1,2 @@
+select encounter_id, CODE, DIAGNOSIS_RANK, code_type, condition_type
```

Comment:
Nitpick: Just wondering why the case is different. It would look better if these were all lowercase. Same goes for the other models.